README.md

```diff
@@ -34,17 +31,37 @@ To get the server running locally:
 ## 2️⃣ Endpoints
 
 #### Register New User
-POST to https://get2it.herokuapp.com/api/auth/register
+#### **POST** to *https://get2it.herokuapp.com/api/auth/register*
```

Comment:
address is: https://get2itpt9.herokuapp.com/api/auth/register

Reply:
Fixed all of these links